README.md

```diff
@@ -22,19 +22,19 @@ _Some similar features might have been implemented/merged_
 - Yarn/npm in favour of bower
   - Load only moment.js without languages (Languages are included in translations)
   - unneeded components removed (es6-shim, tablesort, numeraljs, leaflet-providers, jshashes)
-- RBush v2 - performance boost in last versions (Positions labels and clients on the map)
+- RBush v2 - performance boost in last versions (positions, labels and clients on the map)
 - Ruby dependency removed
 - FixedCenter is required
-- Sass-lint, scss and variables rewritten for easy customization/adjustments
+- Sass-lint, scss and variables rewritten for easy customizations/adjustments
```

Comment:
If you use a slash there should be both plural or both singular.
or we use "for easy customization and adjustments"